joomla/joomla-cms
# [4.0] Remove language options on non multilingual site

Currently we have a column for the language on all list views and a field to select the language on all edit views

This PR removes them when you are on a non multilingual site. This simplifies the UI and removes a useless field when creating content

### Parts
- [x] Lists
- [x] Search tools

## After PR and non multilingual
<img width="704" alt="screenshotr10-20-00" src="https://user-images.githubusercontent.com/1296369/28110930-bae13ee4-66eb-11e7-8022-2ad898c8b92d.png">

<img width="345" alt="screenshotr10-20-55" src="https://user-images.githubusercontent.com/1296369/28110956-cf470332-66eb-11e7-850b-e956aa011e9c.png">

## Before PR and After PR and multilingual
<img width="716" alt="screenshotr10-22-36" src="https://user-images.githubusercontent.com/1296369/28111041-1449186c-66ec-11e7-9936-f0e1898bfa1c.png">
<img width="291" alt="screenshotr10-22-46" src="https://user-images.githubusercontent.com/1296369/28111042-14516e86-66ec-11e7-99cf-feecd02e95b6.png">

## Test instructions
You don't need to install multiple languages. It is enough to enable the System - Language Filter plugin.
When enabled you will have language columns, sorts and filters AND language fields on item creation
When not enabled you never see the language columns

## Thanks
@infograf768 @mbabker @wilsonge
